docs/metrics/meteor.md

```diff
@@ -0,0 +1,101 @@
+---
+search:
+  exclude: true
+---
+
+# METEOR
+
+METEOR (**M**etric for **E**valuation of **T**ranslation with **E**xplicit **OR**dering) is a widely recognized and vital metric used in natural language processing. Originally developed for machine translation workflows, it is used to measure the quality of candidate texts against reference texts for many different workflows. Though it is an n-gram based metric, it goes beyond traditional methods by factoring in elements such as precision, recall, and order to provide a comprehensive measure of text quality. For an in-depth justification behind METEOR's design choices, feel free to check out the [original paper](https://www.cs.cmu.edu/~alavie/METEOR/pdf/Banerjee-Lavie-2005-METEOR.pdf).
+
+## Implementation Details
+
+We define METEOR as the product of two components - the Unigram Precision / Recall Harmonic Mean, and Word Order Penalty. That is,
+
+$$
+\text{METEOR} = \underbrace{\text{FMean}}_{\text{Harmonic Mean of Unigram Precision/Recall}} * \underbrace{(1 - \text{Penalty})}_{\text{Word Order Penalty}}
+$$
+
+To understand the formula, let's break down each component into their respective parts.
+
+
+??? info "FMean: Harmonic Mean of the Unigram Precision / Recall"
+    This is defined as
+
+    $$
+    FMean = \frac{10PR}{R + 9P}
+    $$
+
+    where P represents the unigram precision, and R represents the unigram recall. Here's a recap of [precision](precision.md) and [recall](recall.md). Notice that most of the weight is placed on the recall component by design – this allows METEOR to prioritize the coverage of essential keywords in the candidate text.
+
+??? info "Penalty: Word Order Penalty"
+    Since the FMean is based on unigram precision and recall, to take into account longer sequences, METEOR has a penalty factor to alleviate this weakness and enforce an order on the candidate sentence.
+
+    First, the  unigrams in the candidate text that are mapped to unigrams in the reference text are grouped in such a way that there exists the fewest number of *chunks*, where each chunk consists of adjacent unigrams. Our penalty factor is then defined as:
+
+    $$
+    Penalty = 0.5 \times \frac{\text{# of Chunks}}{\text{# of Unigrams Matched}}
+    $$
+
+    For example, if our candidate sentence was "the president spoke to the audience" and our reference sentence was "the president *then* spoke to the audience", there would be two chunks – "the president" and "spoke to the audience" – and 6 unigrams matched. Notice that as the chunks decrease, so does the penalty, which results in a higher METEOR score. This is quite intuitive as a lower number of chunks translates to an enforced order and better alignment with the reference text.
+
+## Examples
+
+**Candidate**: Joe likes to eat apples for lunch. <br>
```

Comment:
Sorry for nitpicking @billbai0102 but can you change these sentences in a similar format as BLEU or ROUGE docs?

Reply:
Example has been changed